src/lexer.c

```diff
@@ -52,6 +52,39 @@ void ListSymbols()
 		free(str);
 		j++;
 	}
+	free(p);
+}
+
+char * Symbols_Str()
+{
+	int j=0, ln=0;
+	size_t slen=0;
+	char *str, *p;
+	char *line=calloc(0, 1);
+	p=Unicode2Utf8(0x25CC); /*dotted circle for combining marks */
+	j=0;
+	while(Symbols[j].name)
+	{
+		str=Unicode2Utf8(Symbols[j].unicode);
+		if(IsCombiningMark(Symbols[j].unicode))
+		{
+			slen=(size_t)snprintf(NULL, 0, "%s %s:%s%s;", Symbols[j].name, p, str, p);
+			line=realloc(line, ln+slen+1);
```

Comment:
Repeatedly calling `realloc` like this (as opposed to iteratively doubling the size) is an O(n^2) algorithm, no?

Reply:
Yes, you're right (and I was thinking to avoid `strlen` in for the same reason 😅).

Reply:
Solved.